src/safeds/data/tabular/containers/_table.py

```diff
@@ -1774,16 +1774,31 @@ def plot_correlation_heatmap(self) -> Image:
         """
         only_numerical = self.remove_columns_with_non_numerical_values()
 
-        fig = plt.figure()
-        sns.heatmap(
-            data=only_numerical._data.corr(),
-            vmin=-1,
-            vmax=1,
-            xticklabels=only_numerical.column_names,
-            yticklabels=only_numerical.column_names,
-            cmap="vlag",
-        )
-        plt.tight_layout()
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                message=(
+                    "Attempting to set identical low and high xlims makes transformation singular; automatically"
```

Comment:
Instead of using to separate filters, you can also use a regex here (that matches both the text with `xlim` and `ylim`). Makes it easier to read.